lte/gateway/c/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -1132,8 +1132,6 @@ int s1ap_handle_ue_context_release_command(
 
     if (ue_context_release_command_pP->cause == S1AP_IMPLICIT_CONTEXT_RELEASE ||
         ue_context_release_command_pP->cause == S1AP_SCTP_SHUTDOWN_OR_RESET ||
-        ue_context_release_command_pP->cause ==
```

Comment:
Why is this check being removed?

Reply:
The check is removed as release_cause,S1AP_INITIAL_CONTEXT_SETUP_TMR_EXPRD  is removed